### DISCUSSION TITLE
对话经常丢失上文记忆

### DISCUSSION THREAD

**Discussion by youheiheia** [TBXark/ChatGPT-Telegram-Workers#161] [Q&A]
works kv的储存显示6kb/1gb但是好像能储存的最多就6kb了，点进去查看空间记录发现对话记录只能保存几句话，请问这是cloudflare的设置有问题吗还是代码有限制。

**Comment by tbxark** [comment]
有历史记录长度和token数量限制。所以每次都会裁剪历史长度。历史长度过长会导致workers处理超时。所以也不能存储太长。
